tcf_website/templates/base/bugform.html

```diff
@@ -0,0 +1,50 @@
+{% load static %}
+
+{% block styles %}
+<link rel="stylesheet" href="{% static 'common/form.css' %}" />
+{% endblock %}
+
+<!-- found a bug form pop up (modal) -->
+<div class="modal fade" id="bugModal" tabindex="-1" role="dialog" aria-labelledby="bugModal" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="bugModal">Report a Bug </h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <form class="needs-validation" id="bugform" novalidate>
+          <div class="form-row">
+              <label for="urlField">URL</label>
+              <input type="text" class="form-control" id="urlField" placeholder="thecourseforum.com" value="thecourseforum.com" required>
+              <div class="valid-feedback">
+                Looks good!
+              </div>
```

Comment:
Where do these divs (the one here and the one for the description field) show up in the form? I can't figure out where to find them

Reply:
They only show up after the form has been validated and has not been submitted yet (see second screenshot). If everything is good, the form will just submit and close itself.

Reply:
Gotcha, cool